Source/SelfService/Web/index.ejs

```diff
@@ -7,6 +7,7 @@
     <title>
         <%- htmlWebpackPlugin.options.metadata.title %>
     </title>
+    <link rel="shortcut icon" href="{%=o.htmlWebpackPlugin.files.favicon%}" >
```

Comment:
It looks like the HtmlWebpackPlugin had updated behaviour, and this is not needed anymore. It also has the wrong syntax, but that isn't relevant.

have a look at the comments on this SO answer: https://stackoverflow.com/questions/35239733/webpack-html-webpack-plugin-load-favicons-in-template#comment86983431_49595058